test/test.hh

```diff
@@ -88,87 +88,89 @@ namespace test
 
 #else
```

Comment:
Can you also define generating routines for:
- `TEST_CHECK_EQUAL`
- `TEST_CHECK_RELATIVE_ERROR_C`